build.gradle.kts

```diff
@@ -51,21 +52,14 @@ subprojects {
 
   // Only apply if the project has the kotlin plugin added:
   plugins.withType<KotlinPluginWrapper> {
-    val compileKotlin by tasks.getting(KotlinCompile::class) {
```

Comment:
I also changed the syntax of this KotlinCompile configuration to avoid duplication and keep things more consistent.